great-idea/index.html

```diff
@@ -17,6 +17,111 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
+	  
+	   <header>
+ 	<section class="section1">
+	  <nav>
+ <a href="#">Services</a>
+  	<a href="#">Product</a>
+  	<a href="#">Features</a>
+	<a href="#">About</a>
+  	<a href="#">Contact</a>
+  	
+	</nav>
+  
+
+  <img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
+	  </section>
+  
+	  <section class="section2">
+	  <h1 class="main-header"> 
+ 
+		  <div class="innovation">Innovation</div>    
+		<div class="on">On</div>
+		
+	  	   <div class="demand">Demand</div>
```

Comment:
I might have mentioned this before, but splitting the header text into three different dividers really hurts semantics. A better approach would be something like this:

```
<h1>
  Innovation<br>
  On<br>
  Demand
</h1>
```